### DISCUSSION TITLE
Add embeddable talk detail view

### DISCUSSION THREAD

**Discussion by fkusei** [pretalx/pretalx#1188] [Ideas]
It would be nice if the 'talk detail' page could be embedded into another site, similar to embedding a schedule view.

This would allow conference websites to add some talks to other pages, for example to explicitely promote special speakers.

**Comment by rixx** [comment]
Sure, that sounds like a nice idea! We won't be able to prioritise it ourselves right now, but if anybody wants to work on it, feel free to submit a pull request. 

This would need to include:

- widget generation including regenerating whenever event colour changes
- linking to widget embeds on talk detail pages in an unobtrusive way
- mentioning the talk widget on the schedule widget page
